### PR TITLE
[wip] violation summary made mandatory

### PR DESCRIPTION
**Pull request by dwechsle**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/176)

## What does this change?
adds requirement for violation summary freeform field
still needs text edit for required field error message
### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
@alexsoble Figured this one out:
Set 'form_novalidate: True` in views.py to turn off client-side validation

**Comment by LindsayYoung** [comment]
The auto generated default text gets fixed in https://github.com/usdoj-crt/crt-portal/pull/173

File: crt_portal/cts_forms/views.py [hunk 1]
```diff
@@ -96,7 +96,8 @@ def get_context_data(self, form, **kwargs):
 
         if current_step_name == 'Details':
             context.update({
-                'page_note': 'Continued'
+                'page_note': 'Continued',
+                'form_novalidate': True,
```
Comment (el-mapache) [inline crt_portal/cts_forms/views.py]: I use this as well in my location PR, is it worth just setting it globally at this point? I think every form we have with validation is going to use it?

Comment (LindsayYoung) [reply crt_portal/cts_forms/views.py]: That's a great point- I can make that tweak

**Pull request merged**